### ISSUE TITLE
ML-Benchmark prediction submission: DeepESD-IFCAv1-orog

### ISSUE THREAD

**Issue by jgonzalezab** [WCRP-CORDEX/ml-benchmark#29]
### Institution ID

IFCA

### Institution Name

Instituto de Física de Cantabria (IFCA), CSIC-Universidad de Cantabria

### Emulator identifier (`emulator_id`)

DeepESD-IFCAv1-orog

### Emulator Description

The architecture consists of three convolutional layers with 50, 25, and 1 kernels, respectively, each using a ReLU activation function. The output of the final convolutional layer is flattened and passed through a fully connected layer that maps the resulting latent representation to the final prediction. Mean Squared Error (MSE) is used as the loss function for temperature, while precipitation is trained using the Asymmetric (ASYM) loss function defined in [1]. Input data are standardized at the grid-point level using the mean and standard deviation of the predictors corresponding to each training experiment. Models are trained for a variable number of epochs, determined via an early-stopping strategy based on a randomly split validation dataset.

Orography is appended to the output of the final convolutional layer, allowing the fully connected layer to incorporate both the learned features and the orographic information when computing the prediction.

[1] Doury, A., Somot, S., & Gadat, S. (2024). On the suitability of a convolutional neural network based RCM-emulator for fine spatio-temporal precipitation. Climate Dynamics, 62(9), 8587-8613.

### Hardware and Training Details

Models are trained on a single V100 GPU due to cluster availabilit, however, the GPU is underutilized. Each epoch takes approximately 0.5-1 second, and training typically runs for 200–600 epochs, depending on early-stopping criteria.

### Stochastic/Probabilistic Output

no

### Reference URL

https://journals.ametsoc.org/view/journals/aies/4/4/AIES-D-24-0121.1.xml

### Repository for Reproducibility

https://github.com/jgonzalezab/deepESD_CORDEX-ML-Bench
